data/MangoHud.conf

```diff
@@ -79,7 +79,10 @@
 gpu_stats
 # gpu_temp
 # gpu_junction_temp
-# gpu_core_clock
+gpu_core_clock
+gpu_nvdec_clock
+gpu_nvenc_clock
+gpu_vic_clock
```

Comment:
wdym? like enable all of them if only the `gpu_core_clock` param is enabled? I thought the granularity would be nice since not all users will want to waste space showing all of them if they only want one.

Reply:
I might not be understand the structure correct.
Can all 3-4 of these be monitored at the same time?

Reply:
these are different hardware units. their clocks are different and independent

Reply:
So I fully understand, one device will only have one of these?
In that case I don't see a point in the granularity

Reply:
you do not understand. no

these are different hardware engines on the same SOC. nvdec (nvidia decode engine), nvenc (nvidia encode engine), and vic (nvidia image compositor) are all present on the same SOC.

example during chromium hardware accelerated video decode and playback:

![Screenshot_20240528_133546](https://github.com/flightlessmango/MangoHud/assets/28281419/eb057fef-bec6-448b-9b1d-3718cf5555f5)